snap/info.go

```diff
@@ -187,16 +187,26 @@ func (s *Info) CommonDataDir() string {
 	return filepath.Join(dirs.SnapDataDir, s.Name(), "common")
 }
 
-// DataHomeDir returns the per user data directory of the snap.
-func (s *Info) DataHomeDir() string {
+// DataHomeGlob returns the per user data directory of the snap.
+func (s *Info) DataHomeGlob() string {
 	return filepath.Join(dirs.SnapDataHomeGlob, s.Name(), s.Revision.String())
 }
 
-// CommonDataHomeDir returns the per user data directory common across revisions of the snap.
-func (s *Info) CommonDataHomeDir() string {
+// RootUserDataDir returns the user data dir for the root user
+func (s *Info) RootUserDataDir() string {
```

Comment:
It is like @kyrofa said, the home glob is using /home but /root is special. Its a bit ugly, I much agree.